ITechart.Patterns/Singleton/SingletonUsage.cs

```diff
@@ -13,24 +13,35 @@ class SingletonUsage
         public static void UseSingleton()
         {
             Console.WriteLine("First access:");
-            using (AccessToDB access = AccessToDB.GetAccess())
-            {
-                PeopleOuput(access);
+            var context = GetContext();
+            var people = context.People
+                        .ToList()
+                        .Select(person => new Person()
+                        {
+                            Id = person.Id,
+                            Name = person.Name,
+                            Age = person.Age
+                        })
+                        .ToList();
+            PeopleOuput(people);
 
-                Console.WriteLine("\nTrying to another access while singleton instance exists:");
-                using (AccessToDB anotherAccess = AccessToDB.GetAccess()) { }
-            }
+            Console.WriteLine("\nTrying to another access while singleton instance exists:");
+            var context2 = GetContext();
+            var agesSum = context.People.Sum(item => item.Age);
```

Comment:
"context2" variable should be used here instead of "context", currently "context2" have no usages.